src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c

```diff
@@ -1307,13 +1314,29 @@ static int get_temp_by_m_tag(struct xocl_xmc *xmc, char *m_tag)
 /* Runtime clock scaling sysfs node */
 static bool scaling_condition_check(struct xocl_xmc *xmc, struct device *dev)
 {
+	if (xmc->sc_presence) {
+		//Feature present bit may configured each time an xclbin is downloaded,
+		//or following a reset of the CMC Subsystem. So, check for latest
+		//status every time.
+		xmc->cs_on_ptfm = false;
+		xmc->runtime_cs_enabled = false;
+		u32 reg = READ_REG32(xmc, XMC_HOST_NEW_FEATURE_REG1);
```

Comment:
Please remove all .o files and rerun make to avoid any warning.

/proj/xsjhdstaff2/davidzha/XRT_zocl/src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/../subdev/xmc.c:1323:3: warning: ISO C90 forbids mixed declarations and code [-Wdeclaration-after-statement]
   u32 reg = READ_REG32(xmc, XMC_HOST_NEW_FEATURE_REG1);
   ^
  LD [M]  /proj/xsjhdstaff2/davidzha/XRT_zocl/src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/xclmgmt.o